go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gruntwork-io/terratest
 
-go 1.21.1
+go 1.21.0
```

Comment:
I was wondering if can be specified `go 1.21`

Reply:
Yes, it's possible. There are multiple references in https://go.dev/blog/using-go-modules